### DISCUSSION TITLE
How to upgrade

### DISCUSSION THREAD

**Discussion by prnzngr** [librenms/docker#421] [Q&A]
Since 24.1 is already released, how can I update a docker installation to latest?
daily.sh is not working.

**Comment by Wenkow** [comment]
I would run 

```
 docker-compose down && docker-compose pull && docker-compose up -d
```
in the directory containing the compose file

**Comment by prnzngr** [comment]
Unfortunatelly it's still 23.11 downloading

**Comment by Wenkow** [comment]
Because it is today still the latest container image available.

[https://hub.docker.com/r/librenms/librenms/tags](https://hub.docker.com/r/librenms/librenms/tags)
